src/engine/test/source/builder/builders/opBuilderHelperAppendSplitString_test.cpp

```diff
@@ -0,0 +1,106 @@
+#include <any>
+#include <gtest/gtest.h>
+#include <vector>
+
+#include <baseTypes.hpp>
+
+#include "opBuilderHelperMap.hpp"
+
+using namespace base;
+namespace bld = builder::internals::builders;
+
+TEST(OpBuilderHelperAppendSplitString, Builds)
+{
+    auto tuple = std::make_tuple(std::string {"/field"},
```

Comment:
I would add to every non-variant variable the modifier `const`. This comment applies to every file.